include/geode/mesh/core/regular_grid_solid.h

```diff
@@ -94,7 +94,7 @@ namespace geode
 
         local_index_t get_nb_polyhedron_facets( index_t /*unused*/ ) const final

```

Comment:
why don't using nb of cell neighbor as a name... nb_of_polyhedron_facets is not really explicit for me it should be six for every cells no?.

Reply:
To discuss meeting in a meeting. Currently, this is mandatory because of the SolidMesh inheritance. I do not like to use `cell` in the general case because a cell is supposed to be in a structured mesh. So using cell for a tetrahedron is not really right...